src/daemon.cpp

```diff
@@ -195,7 +195,7 @@ static void
 announceDeviceRemoval(Input& input)
 {
 	NE2Packet pkt = {};
-	::memset(&pkt, 0, sizeof(pkt));
+	::memset((void *) &pkt, 0, sizeof(pkt));
```

Comment:
I'm currently building with -Wold-style-cast on with clang. While I don't mind dropping that much, it would be nicer to stay consistent here.
Also you have a space here and in main.cpp, but not in reader.cpp, please cast without the space.

But I wonder, which compiler+version actually complains about this?